docs/js.md

```diff
@@ -0,0 +1,162 @@
+### [Primative: Null](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null)
+
+* _symbol_: `null`
+* _pronunciation_: "null", "null value", "is null"
+* _examples_:
+```javascript
+null // how exciting
+```
+
+### [Primative: Boolean](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)
+
+*_symbol_: 'true' 'false'
+*_pronunciation_"boo-lee-an"
+*_examples_:
+```javascript
+50 < 100 == true
+50 > 100 == false
+
+('meh' != meh) == true
+```
+
+### [Primative: Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number)
+
+*_symbol_: 'The numeric value of the object being created.'
+*_pronunciation_"number" "integer"
+*_examples_:
+```javascript
+1
+2
+3
+10
+```
+
+### [Primative: String](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String)
+
+*_symbol_: 'a sequence of characters'
+*_pronunciation_"string" "123456789" "abcdefghijk"
+*_examples_:
+```javascript
+'I like big cats' + '\n and I cannot lie'
+```
+
+### [Built in Values: Infinity](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+
+*_symbol_: 'Infinity'
+*_pronunciation_: "in-fin-it-ee"
+*_examples_:
+```javascript
+3 * infinity == infinity
+```
+
+### [Built in Values: NaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+
+*_symbol_: 'NaN'
+*_pronunciation_: "not a number", "nahn"
+*_examples_:
+```javascript
+"cat" == NaN
+```
+
+### [Built in Values: Undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity)
+
+*_symbol_: 'undefined'
+*_pronunciation_: "un-defined"
+*_examples_:
+```javascript
+
+```
+
+
+### [Operator: Addition](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Addition_(.2B))
+
+* _symbol_: `+`
+* _pronunciation_: "plus", "added to"
+* _examples_:
+```javascript
+1 + 2 + 3 // 6, of course
+'1' + 2 + 3 // '123', obviously... WTF?
+```
+
+### [Operator: Modulus](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: `%`
+*_pronunciation_: "mod-u-lus"
+*_example_:
+```javascript
+43 % 10
+Returns 3
+```
+
+### [Operator: Increment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: '++'
+*_pronunciation_: "increment"
+*_example_:
+```javascript
+12++ == 13
+```
+
+### [Operator: Decrement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#Arithmetic_operators)
+
+*_symbol_: '--'
+*_pronunciation_: "decrement"
+*_example_:
+```javascript
+12-- == 11
+```
```

Comment:
For both the increment and decrement operator, you'll run into some issues running those examples. These are both _Assignment Operators_ and, as such, can only be used with values stored in variables, e.g.

```javascript
var A = 10, B = -10;

console.log(--A); // prints 9
console.log(B++); // prints -10
console.log(B); // prints -9...?
```